app/ports/management/commands/update-portinfo.py

```diff
@@ -0,0 +1,57 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from parsing_scripts import git_update
+from ports.models import Port, LastPortIndexUpdate
+
+
+class Command(BaseCommand):
+    help = "Populates the database with Initial data from portindex.json file"
+
+    def add_arguments(self, parser):
+        parser.add_argument('new',
+                            nargs='?',
+                            default=False,
+                            help="Define a commit till which the update should be processed")
+        parser.add_argument('old',
+                            nargs='?',
+                            default=False,
+                            help="Not recommended. Helps you provide a commit from which update should start")
+
+    def handle(self, *args, **options):
+        # Fetch from rsync
+        Port.RsyncHandler().sync()
+
+        # Open the file
+        data = Port.RsyncHandler().open_file()
+
+        # If no argument is provided, use the commit-hash from JSON file:
+        if options['new'] is False:
+            new_hash = data['info']['commit']
+        # If argument is provided
+        else:
+            new_hash = options['new']
+
+        # Using the range of commits, find the list of ports which have been updated
+        # options['old'] will be false if no argument is provided
```

Comment:
If no argument for `old_commit` was provided, `options['old_commit']` will be `None` and the code will use the last commit from the previous update.

(Just suggestion.)